lighthouse/config/schemas.py

```diff
@@ -11,6 +11,25 @@
         "type": "string",
         "check_with": "plate_events_dependent_parameters",
     },
+    "user_id": {
+        "required": True,
+        "type": "string",
+    },
+    "robot": {
+        "required": True,
+        "type": "string",
+    },
+    "event_wh_uuid": {
+        "type": "string",
+        "unique": True,
+        "readonly": True,
+    },
+    "errors": {
+        "type": "dict",
```

Comment:
Dict or list?

Reply:
Is a dict because it has the list of errors for each property of the event (PlateEvent.errors())